dll/win32/kernel32/client/proc.c

```diff
@@ -486,6 +486,13 @@ BaseProcessStartup(PPROCESS_START_ROUTINE lpStartAddress)
                                &lpStartAddress,
                                sizeof(PPROCESS_START_ROUTINE));
 
+	#if defined(__i386__) 
+	    /*Set EBX = PEB */
+	    asm volatile (".intel_syntax noprefix");
```

Comment:
Switching to Intel syntax can do unexpected things. Please just write the instruction you need in AT&T.
Also, you need an MSVC version of this.

Reply:
Will do :) .